setup.py

```diff
@@ -31,7 +31,7 @@ def run(self):
     packages=find_packages(),
     url="https://github.com/ucgmsim/IM_calculation",
     description="IM calculation code",
-    install_requires=["numpy>=1.14.3", "numba>=0.43.1", "Cython", "pandas"],
+    install_requires=["numpy>=1.14.3", "numba>=0.43.1", "Cython", "obspy", "pandas"],
```

Comment:
Where does im_calc import obspy? I'm not surprised if it does - just can't recall where.

Reply:
```
File "/tmp/pip-req-build-zqcy8hon/IM_calculation/scripts/A_KonnoMatricesComputation.py", line 4, in <module>                          
      from obspy.signal.konnoohmachismoothing import calculate_smoothing_matrix                                                           
  ModuleNotFoundError: No module named 'obspy'                                                                                            
  ----------------------------------------                                                                                                
  ERROR: Failed building wheel for IM-calc
  ```